lib/autocomplete.js

```diff
@@ -0,0 +1,34 @@
+const TypeMapping = require('../lib/type-mapping');
+
+module.exports = (elastic, queryParams, cb) => {
+  const must = [{
+    match_phrase_prefix: {
+      summary_title_text: { query: queryParams.q }
+    }
+  }];
+
+  if (queryParams.type) {
+    must.push({
+      term: { 'type.base': TypeMapping.toInternal(queryParams.type) }
+    });
+  }
+
+  const searchOpts = {
+    searchName: 'autocomplete',
+    index: 'smg',
+    size: queryParams.size || 3,
+    _source: ['summary_title'],
+    body: {
+      query: {
+        bool: {
+          must,
```

Comment:
I think you don't need to precise the property must if it's empty, but is' more a stylish issue